testframework/build.gradle

```diff
@@ -19,9 +20,17 @@ repositories {
     }
 }
 
+configurations {
+    localClasspath
+    compileClasspath.extendsFrom(localClasspath)
```

Comment:
Why not add to `compileClasspath` directly?

Reply:
Because it can't be consumed.

Reply:
Add them to `compileOnly` then.